### PR TITLE
fix(TCOMP-553):fix windows service startup failure

### PR DESCRIPTION
**Pull request by sgandon**
**What is the current behavior?** (You can also link to an open issue here)
Service fails to start on windows cause the classpath is too long

**What is the new behavior?**
Now it starts. The classpath as been moved from the start.bat to the MANIFEST.MF _Class-Path_ key.

**Please check if the PR fulfills these requirements**

- [x] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/544/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/544/)

File: services/components-api-service-rest/assembly/config.xml [hunk 1]
```diff
@@ -7,20 +7,23 @@
...
-            <includes>
-                <include>*.*</include>
-            </includes>
-        </fileSet>
-        <fileSet>
-            <directory>${project.basedir}/scripts</directory>
-            <outputDirectory>${file.separator}</outputDirectory>
+		<!-- configuration files -->
+		<fileSet>
+			<directory>${project.basedir}/target/classes</directory>
```
Comment (gonchar-ivan) [inline services/components-api-service-rest/assembly/config.xml]: maybe it is required to replace '/' with ${project.basedir} here too

File: services/components-api-service-rest/assembly/no-arch.xml [hunk 2]
```diff
@@ -6,6 +6,16 @@
 	<formats>
 		<format>zip</format>
 	</formats>
+	<dependencySets>
+		<!-- place executable jar in root -->
+		<dependencySet>
```
Comment (gonchar-ivan) [inline services/components-api-service-rest/assembly/no-arch.xml]: code formatting

File: services/components-api-service-rest/pom.xml [hunk 3]
```diff
@@ -229,6 +208,32 @@
...
+								<exclude>**/settings.xml</exclude>
+							</excludes>
+							<archive>
+								<manifest>
+									<addClasspath>true</addClasspath>
+									<classpathPrefix>.m2/</classpathPrefix>
+									<classpathLayoutType>repository</classpathLayoutType>
+									<useUniqueVersions>false</useUniqueVersions><!-- to avoid time stamped snapshots -->
+									<mainClass>org.talend.components.service.rest.Application</mainClass>
+								</manifest>
```
Comment (gonchar-ivan) [inline services/components-api-service-rest/pom.xml]: I saw some strange entries in MANIFEST file when I built rest-service
Implementation-URL: http://projects.spring.io/spring-boot/daikon-service/service-parent/components-api-service-rest/
Implementation-Vendor: Pivotal Software, Inc.

url is not working 
and seems Pivotal is not correct value for vendor.

File: services/components-api-service-rest-all-components/pom.xml [hunk 4]
```diff
@@ -3,13 +3,15 @@
...
 
+
 	<parent>
-		<groupId>org.talend.components</groupId>
-		<artifactId>components-parent</artifactId>
-		<version>0.19.0-SNAPSHOT</version>
-		<relativePath>../../components-parent/pom.xml</relativePath>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>1.5.1.RELEASE</version>
```
Comment (gonchar-ivan) [inline services/components-api-service-rest-all-components/pom.xml]: I can't build all-components service.
Please add repository in plugin repositories:
```
		<repository>
			<id>releases</id>
			<url>https://artifacts-oss.talend.com/nexus/content/repositories/TalendOpenSourceRelease/</url>
		</repository>
```

Also I don't know the purpose of talend-update repository. Do we need it here?

**Comment by gonchar-ivan** [comment]
checked start scripts. Everything works: both .bat and .sh for both bare and all-components services

File: services/components-api-service-rest-all-components/assembly/no-arch.xml [hunk 5]
```diff
@@ -5,10 +5,20 @@
     <formats>
         <format>zip</format>
     </formats>
+	<dependencySets>
+		<!-- place executable jar in root -->
+		<dependencySet>
```
Comment (sgandon) [inline services/components-api-service-rest-all-components/assembly/no-arch.xml]: done

File: services/components-api-service-rest-all-components/assembly/no-arch.xml [hunk 6]
```diff
@@ -5,10 +5,20 @@
...
+            <includes>
+                <include>org.talend.components:components-api-service-rest:jar:${project.version}</include>
+            </includes>
+        </dependencySet>
+	</dependencySets>
     <fileSets>
         <!-- configuration files -->
         <fileSet>
-            <directory>${project.basedir}/target/external-resources/config</directory>
+            <directory>${project.basedir}${file.separator}target${file.separator}external-resources/config</directory>
```
Comment (sgandon) [inline services/components-api-service-rest-all-components/assembly/no-arch.xml]: done

File: services/components-api-service-rest/assembly/config.xml (hunk 1 above)
Comment (sgandon) [inline services/components-api-service-rest/assembly/config.xml]: done (with ${file.separator})

File: services/components-api-service-rest/assembly/no-arch.xml (hunk 2 above)
Comment (sgandon) [inline services/components-api-service-rest/assembly/no-arch.xml]: done

File: services/components-api-service-rest/pom.xml (hunk 3 above)
Comment (sgandon) [inline services/components-api-service-rest/pom.xml]: done

File: services/components-api-service-rest-all-components/pom.xml (hunk 4 above)
Comment (sgandon) [inline services/components-api-service-rest-all-components/pom.xml]: nice catch !

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/568/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/568/)

**Pull request merged**